maintenance/zero_downtime_migration.rst

```diff
@@ -0,0 +1,167 @@
+.. _zero_downtime_migration:
+
+Zero-downtime Migration
+===============================
+
+Firely Server v6 introduces an enhanced MongoDB database migration process. In contrast to the previous migration method that required system downtime, this new approach enables zero-downtime migration. This allows for continuous operation of your existing Firely Server instance while the data migration to the new schema proceeds.
+
+.. note::
+   This migration process applies to the *vonkdata* database. 
+   
+   If you also use MongoDB as the **admin** database, you will need to initialize a new *vonkadmin* database from scratch by specifying a different connection string for the admin database than your previous installation.
+
+.. warning::
+   Please note that any snapshot produced by the bulk data export functionality will not be available after the migration.
+
+.. important::
+   Bulk data export data is excluded from the migration process.
+
+Architecture Overview
+---------------------
+
+The following diagram illustrates the architecture and components involved in the migration process:
+
+.. image:: migration-diagram.svg
+   :alt: Migration Diagram
+
+The architecture consists of:
+
+* An existing database containing the source data
+* The current Firely Server instance maintaining normal operations during migration
+* A new target database instance initialized empty
+* A migration tool operating asynchronously to transform and transfer data
+* A new Firely Server instance configured for the target database
+* A reverse proxy managing request routing between instances
+
+.. note::
+   During the migration period, both database instances operate concurrently.
```

Comment:
Maybe also note that the migration is in :ref:`one direction <https://en.wikipedia.org/wiki/One_Direction>` only. So although even both *Server* instances could work concurrently, changes from the new instance are not synced back to the old instance.

Reply:
haha, what's this link? :)

Reply:
I will add a note on this nonetheless

Reply:
@eugene-blokhin : The link was a joke ;-)